### PR TITLE
F#362 add examples for hand finder

### PR DESCRIPTION
**Pull request by VahidAminZ**
#362

### CODE REVIEW

**Comment by toliver** [comment]
The example script should have the executable flag. Haven't you run it with rosrun?

**Comment by VahidAminZ** [comment]
I tested it with pycharm. I will change it.

**Comment by toliver** [comment]
I get an exception when no hand is running. The python library should behave correctly in that case (same as you did for the cpp one).

Add a test to check that maybe?

File: sr_hand/launch/gazebo/loaders/hand_controllers.launch [hunk 1]
```diff
@@ -1,9 +1,12 @@
 <launch>
   <arg name="hand_id" default="rh" />
+  <arg name="hand_serial" default="1234"/>
   <arg name="hand_controller_gazebo"
   default="$(find sr_description)/hand/config/$(arg hand_id)_controller_gazebo.yaml" />
   <arg name="hand_effort_controller_gazebo"
   default="$(find sr_description)/hand/config/$(arg hand_id)_effort_controller_gazebo.yaml" />
+  <param name="/hand/mapping/$(arg hand_serial)" value="$(arg hand_id)"/>
+  <param name="/hand/joint_prefix/$(arg hand_serial)" value="$(arg hand_id)_"/>
```
Comment (toliver) [inline sr_hand/launch/gazebo/loaders/hand_controllers.launch]: This parameter is loaded here and also in sr_hand/launch/gazebo/gazebo_hand.launch. Shouldn't it be only in one place?

File: sr_robot_launch/launch/right_srhand_ur10arm.launch [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <launch>
   <!-- This launch file is for a right UR10 arm and shadow hand -->
+  <arg name="hand_id" default="rh" />
```
Comment (toliver) [inline sr_robot_launch/launch/right_srhand_ur10arm.launch:3]: Weird diff glitch? This seems to be the same in indigo-devel.

Comment (VahidAminZ) [inline sr_robot_launch/launch/right_srhand_ur10arm.launch:3]: I do not have this one in indigo-devel.

Comment (toliver) [inline sr_robot_launch/launch/right_srhand_ur10arm.launch:3]: True. It's a mental glitch.

**Pull request merged**